conf/solr/conf/managed-schema

```diff
@@ -142,6 +142,7 @@
     <field name="has_fulltext" type="boolean"/>
     <field name="title" type="text_en_splitting"/>
     <field name="title_suggest" type="text_general" multiValued="false"/>
+    <field name="title_sortable" type="text_title_sort" multiValued="false"/>
```

Comment:
Oh this is what's failing the test; you'll need to add `text_title_sort` to the `type_map` in openlibrary/tests/solr/test_types_generator.py .